queue/QueueRunner.feature

```diff
@@ -5,9 +5,7 @@ Feature: Command and control using a message broker
 
   Scenario: Default client setting
     Then the time to wait for requests is 500ms
-    Then the request queue is "testuser.req"
```

Comment:
The names of the queues (testuser-req) should move to the Background (aka Given) and used to create the queues and initialise the client with them.

Example:
Given I start with a clean broker 
And a request queue named testuser-req 
And a response queue named testuser-resp
And a client for user "testuser" with response queue "testuser-req" and request queue "testuser-resp"

Reply:
One more point we already saw above, on SQS these names are good:

testuser-req 
testuser-resp

but not

testuser.req
testuser.resp

So the tests will not reflect the two implementations i.e. SQS and JMS